charts/crowdsec/Chart.yaml

```diff
@@ -92,4 +92,4 @@ version: 0.9.12
 # This is the version number of the application being deployed. This version number should be
 # incremented each time you make changes to the application. Versions are not expected to
 # follow Semantic Versioning. They should reflect the version the application is using.
-appVersion: v1.6.0
+appVersion: v1.6.1-1
```

Comment:
We found a current issue could we bump the image to `1.6.1-2` hence why we do the rollout